cdap-proto/src/main/java/co/cask/cdap/proto/id/ScheduleId.java

```diff
@@ -117,7 +117,7 @@ public static ScheduleId fromIdParts(Iterable<String> idString) {
   }
 
   @Override
-  protected Iterable<String> toIdParts() {
+  public Iterable<String> toIdParts() {
```

Comment:
If you make this public, then we don't need the helper method in Schedulers any longer